app/experimenter/nimbus-ui/src/components/PageHome/DirectoryTable/index.test.tsx

```diff
@@ -105,14 +105,10 @@ function expectTableCells(testId: string, cellTexts: string[]) {
 }
 
 describe("DirectoryTable", () => {
-  it("renders as expected with default columns", () => {
+  fit("renders as expected with default columns", async () => {
```

Comment:
Re: `it` => `fit`,

What dat about? I'm a bit unclear after https://jestjs.io/docs/api#testonlyname-fn-timeout about which tests will be run with that tweaked. If this `describe("DirectoryTable", ...)` suite has three `it` tests set, but one of them is `fit`, does that mean it will only run the 1 of the 3 tests now?

✅  `fit("renders as expected with default columns", async () => {...}`
❌  `it("renders as expected without experiments", () => {...}`
❌  `it("renders as expected with custom columns", () => {...}`

Reply:
Whoops! Leftover from testing. Good catch.